src/openklant/components/klantinteracties/api/viewsets/klantcontacten.py

```diff
@@ -262,3 +263,21 @@ class ActorKlantcontactViewSet(viewsets.ModelViewSet):
     filterset_class = ActorKlantcontactFilterSet
     authentication_classes = (TokenAuthentication,)
     permission_classes = (TokenPermissions,)
+
+
+@extend_schema(tags=["klantcontact-convenience"])
+@extend_schema_view(
+    create=extend_schema(
+        summary="Maak een KlantContact, Betrokkene en een OnderwerpObject aan.",
+        description="Maak een KlantContact, Betrokkene en een OnderwerpObject aan.",
+    ),
+)
+class KlantContactConvenienceViewSet(mixins.CreateModelMixin, viewsets.GenericViewSet):
+    """
+    Convenience endpoint om in één request een Betrokkene, KlantContact en een OnderwerpObject
+    aan te maken. De aangemaakte objecten worden automatisch aan elkaar gekoppeld.
+    """
+
+    serializer_class = KlantContactConvenienceSerializer
+    authentication_classes = (TokenAuthentication,)
```

Comment:
In think this should be the default for the API but is not part of the ticket :grinning:

Reply:
I assume since the other endpoints use tokenauth, this one should too :p